test/column/formatter.test.tsx

```diff
@@ -110,3 +110,41 @@ describe('Custom formatter component', () => {
     });
   });
 });
+
+describe('formatter focus', () => {
+  const columns: readonly Column<Row>[] = [{ key: 'id', name: 'ID' }];
+  function FormatterTest() {
+    const [inputValue, setInputValue] = useState('');
+    const [rows, setRows] = useState((): readonly Row[] => [{ id: 1 }]);
+
+    function onChange(event: React.ChangeEvent<HTMLInputElement>) {
+      setInputValue(event.target.value);
+      setRows([{ id: 2 }]);
+    }
+
+    return (
+      <StrictMode>
+        <input aria-label="outer-input" value={inputValue} onChange={onChange} />
```

Comment:
Could be a button, focus should remain on the button (if we didn't have the bug)